### PR TITLE
[raz] Auto detect ADLS cluster name from config file

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Support auto-detect for ADLS cluster name along with S3

## How was this patch tested?

- Running unit tests

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: desktop/libs/hadoop/src/hadoop/core_site_tests.py [hunk 1]
```diff
@@ -72,6 +72,10 @@ def test_core_site():
     <name>fs.azure.ext.raz.rest.host.url</name>    
     <value>https://prakashdh67-master10.prakashr.xcu2-8y8x.dev.cldr.work:6082/</value>  
   </property> 
+  <property>
+    <name>fs.azure.ext.raz.adls.access.cluster.name</name>
+    <value>prakashdh67</value>
```
Comment (romainr) [inline desktop/libs/hadoop/src/hadoop/core_site_tests.py:77]: prakashdh67 --> gehue-adls

Comment (romainr) [reply desktop/libs/hadoop/src/hadoop/core_site_tests.py:77]: prakashmowdev1 --> gethue-dev

Comment (romainr) [reply desktop/libs/hadoop/src/hadoop/core_site_tests.py:77]: prakashdh67-master10.prakashr.xcu2-8y8x.dev.cldr.work --> gehue-adls-master

File: desktop/libs/hadoop/src/hadoop/core_site.py [hunk 2]
```diff
@@ -139,7 +140,7 @@ def get_raz_cluster_name():
 
   S3 only. Add check for Azure when supported.
   """
-  return get_conf().get(_CNF_S3A_RAZ_CLUSTER_NAME, '')
+  return get_conf().get(_CNF_S3A_RAZ_CLUSTER_NAME, '') or get_conf().get(_CNF_ADLS_RAZ_CLUSTER_NAME, '')
```
Comment (romainr) [inline desktop/libs/hadoop/src/hadoop/core_site.py:143]: "  S3 only. Add check for Azure when supported." to remove

**Review by romainr — APPROVED** [review]
And while we are there, how about anonymizing more the test data?

File: desktop/libs/hadoop/src/hadoop/core_site_tests.py (hunk 1 above)
Comment (Harshg999) [reply desktop/libs/hadoop/src/hadoop/core_site_tests.py]: Done!

**Pull request merged**